frontend/src/components/AgentStatusBar.tsx

```diff
@@ -61,6 +62,16 @@ function AgentStatusBar() {
   // - Agent is thinking
   // - Agent is ready
   // - Agent is not available
+  useEffect(() => {
+    if (
+      curAgentState === AgentState.AWAITING_USER_INPUT ||
+      curAgentState === AgentState.ERROR ||
+      curAgentState === AgentState.INIT
+    ) {
+      if (document.cookie.indexOf("audio") !== -1) beep();
```

Comment:
How should the user configure this? Can we make this optional in the setting menu of the frontend - and default it to false?

Reply:
@amanape Could you take a look at this?

Reply:
added an icon